ntoskrnl/ex/handle.c

```diff
@@ -1360,7 +1360,11 @@ BOOLEAN ExpKdbgExtHandle(ULONG Argc, PCHAR Argv[])
         }
         else
         {
+#ifdef _WIN64
+            ProcessId = (HANDLE)strtoull(Argv[1], &endptr, 10);
```

Comment:
`strtoulptr`?

Reply:
`strtoulptr()` exists in ntoskrnl/kdbg/kdb_cli.c only (yet).
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=strtoulptr

Reply:
Superseded as part of 253362509ec64a2def87fdc818f3d553f23a20c6.